### PR TITLE
chore: CRS naming, Coraza link typo

### PR DESCRIPTION
**Pull request by M4tteoP**
Hi there, proposing minor changes for the CRS page. 
**Summary** :  
- Adheres to https://github.com/OWASP/www-project-modsecurity-core-rule-set/pull/18 updating the Core rule set project name to `CRS`
- Adds a `]` to properly render Coraza link.

### CODE REVIEW

**Review by jgadsden — DISMISSED** [review]
thanks @M4tteoP , good to have these changes and to get the naming correct

**Comment by jgadsden** [comment]
The spell check failed on 'modcrs' , which can be added to file `.wordlist.txt` if you like @M4tteoP or I cna do it in the next PR
The changes go into draft initially, and promoted to release with next version (which is probably quite soon)

File: draft/11-operations/04-modsecurity-crs.md [hunk 1]
```diff
@@ -26,22 +26,22 @@ permalink: /draft/operations/modsecurity_core_rule_set/
 
 ### 9.4 OWASP CRS
 
-The OWASP CRS[modcrs-project] project is a set of generic attack detection rules
+The [OWASP CRS][crs-project] project, formerly known as Core Rule Set, is a set of generic attack detection rules
```
Comment (M4tteoP) [inline draft/11-operations/04-modsecurity-crs.md:29]: I missed the brackets here, that's why the spellchecker was failing. I tried to run it locally and it looks happy now.

File: draft/11-operations/00-toc.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ Operations generally cover the security practices:
 * [Environment Management][sammoem] such as configuration hardening, patching and updating
 * [Operational Management][sammoom] which includes data protection and system / legacy management
 
-OWASP projects provide the Core Rule Set that is used for both Coraza and ModSecurity web application firewalls,
```
Comment (M4tteoP) [inline draft/11-operations/00-toc.md:30]: Not sure what the difference is between `02-toc.md` and `00-toc.md`, but being both under `/draft`, I changed them. I hope it is all right

**Comment by M4tteoP** [comment]
Thanks @jgadsden for the quick reply. I changed some more occurrences across different pages, extended a sentence to still state the project was named `Core Rule Set`, and (hopefully) fixed the spellchecker.
PTAL, and thanks for your time!

**Comment by jgadsden** [comment]
Superb, thanks @M4tteoP and all checks are passing now
I will go ahead and merge your changes

**Pull request merged**